lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -125,7 +125,7 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
   void finalize();
 
   void setCurrentLoc(IRBuilder &Builder, const SILDebugScope *DS,
-                     Optional<SILLocation> Loc = None);
+                     Optional<SILLocation> Loc);
```

Comment:
Is there a caller left that actually passes None? Otherwise we could drop the Optional.

Reply:
Good call, I'll do that.